src/components/screen-header/ScreenHeader.tsx

```diff
@@ -215,28 +206,15 @@ const useHeaderLayouts = () => {
       setHeaderLayouts((prev) => ({...prev, [element]: layout}));
     };
 
-  const buttonsOnOwnLine = useMemo(() => {
-    const {container, title, leftButton, rightButton} = headerLayouts;
-    if (!container || !title || !leftButton || !rightButton) {
-      return false;
+  const buttonsHeight = useMemo(() => {
+    const {container} = headerLayouts;
+    if (!container) {
+      return 0;
     }
-
-    const leftButtonVisibleWidth = leftButton.height ? leftButton.width : 0;
-    const rightButtonVisibleWidth = rightButton.height ? rightButton.width : 0;
-    const widestButtonWidth = Math.max(
-      leftButtonVisibleWidth,
-      rightButtonVisibleWidth,
-    );
-    const buttonsAndTitleWidth = title.width + widestButtonWidth * 2;
-    const containerWidth = container.width;
-    return buttonsAndTitleWidth > containerWidth - 10;
+    return headerLayouts.container?.height || 0;
```

Comment:
Is setting `0` here the most sensible default?

Reply:
This way we can check if the value is falsy and use the line height if so (i.e. the expression on line 154). Another approach is to make the hook have access to `theme` and return the line height instead of 0 🤔